app/helpers/password.js

```diff
@@ -0,0 +1,8 @@
+const bcrypt = require('bcryptjs');
+
+exports.encryptPassword = async password => {
+  const salt = await bcrypt.genSaltSync(8, 'a');
+  return bcrypt.hashSync(password, salt);
+};
+
+exports.validPassword = (password, comparePassword) => bcrypt.compare(password, comparePassword);
```

Comment:
You are not using password encryption and it is being saved as is in the database.